coderd/coderd.go

```diff
@@ -125,7 +125,10 @@ type Options struct {
 	TemplateScheduleStore *atomic.Pointer[schedule.TemplateScheduleStore]
 	// AppSigningKey denotes the symmetric key to use for signing app tickets.
 	// The key must be 64 bytes long.
-	AppSigningKey []byte
+	AppSigningKey      []byte
+	HealthcheckFunc    func(ctx context.Context) (*healthcheck.Report, error)
+	HealthcheckTimeout time.Duration
+	HealthcheckRefresh time.Duration
```

Comment:
🤨

Reply:
You must be looking at the merge commit or something because that's not in the PR diff.

Reply:
classic github